CONTRIBUTING.md

```diff
@@ -136,3 +136,25 @@ We use swagger to document all of our endpoints and we require developers to ann
 
 - [echo-swagger](https://github.com/swaggo/echo-swagger)
 - Add comments to your API source code, See [Declarative Comments Format](https://github.com/swaggo/swag#declarative-comments-format).
+
+# Testing
+## Postman
+Run postman tests (these test API changes against swagger documentation)
+
+```
+export APIKEY=your_api_key
+./estuary &
+cd tests/postman
+npm install
```

Comment:
maybe should do `npm ci` here instead of `npm install`, to ensure compatibility and avoid overwriting `package.json`

Reply:
actually, just noticed there is no `package-lock.json` committed. Any reason for that? If no then it's good your committing it here but we should make sure any other pipeline code does `npm ci` so it won't break in the future